src/test/java/model/DeadlineTest.java

```diff
@@ -0,0 +1,23 @@
+package model;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class DeadlineTest {
+
+    @Test
+    public void dateParsing_properDate_dateParsedCorrectlyForSaving(){
```

Comment:
You could consider naming the test methods following the coding standards for naming test methods.